src/pyramid_googleauth/__init__.py

```diff
@@ -0,0 +1,15 @@
+from pyramid_googleauth.routes import add_routes
+from pyramid_googleauth.security import GoogleSecurityPolicy
+from pyramid_googleauth.services import GoogleAuthService, SignatureService
```

Comment:
I wonder if these two service classes should be underscored as well, since I don't think they're part of the public interface either